include/Nodes1DProvisioner.hpp

```diff
@@ -55,11 +65,18 @@ class Nodes1DProvisioner {
     Array<double, 2> & get_J();
     Array<double, 2> & get_rx();
 
+    Array<int, 1> & get_Fmask();
+    Array<double, 2> & get_Fx();
+
     Array<int, 2> & get_EToV();
     Array<double, 2> & get_Lift();
   
     Array<int, 2> & get_EToE();
     Array<int, 2> & get_EToF();
+
+    blitzdg::index_vector_type & get_vmapM();
+    blitzdg::index_vector_type & get_vmapP();
+
```

Comment:
Is the user allowed to make changes to the arrays returned by these getters? If not, then you should return a const reference and make the member functions themselves const.

Reply:
Good call, I'll do this. We'll probably have to clean up other places I've done this at some point, too.